homeassistant/components/workday/binary_sensor.py

```diff
@@ -53,8 +56,8 @@
     "DE",
     "Hungary",
     "HU",
-    "Honduras",
-    "HUD",
```

Comment:
Why are you removing [Honduras](https://github.com/dr-prodigy/python-holidays/blob/master/holidays.py#L4007)?

Reply:
Oh I am sorry but it was not in the [list](url) and before removing it I tested it with `HUD` and got the following error:
```log
line 163, in setup_platform
    obj_holidays = getattr(holidays, country)(years=year)
AttributeError: module 'holidays' has no attribute 'HUD'
```
Interestingly enough after testing it again now 'HUD' does not work but `Honduras` does. @fabaff what should I do about that?

Reply:
It's `HND` not `HUD` according the source code.

Reply:
You are correct, did not see [that line](https://github.com/dr-prodigy/python-holidays/blob/a331982e9aef1580ac78883dcb21b0ce2ac6036e/holidays.py#L4011).